src/app/components/top/solicitudes/solicitudes.html

```diff
@@ -104,7 +104,7 @@
                                                     <td *ngIf="!prestacion.solicitud.turno" class="text-center estado"
                                                         (click)="seleccionar(prestacionesEntrada,i)">
                                                         <span class="badge badge-info" *ngIf="prestacion.estados[prestacion.estados.length-1].tipo !== 'rechazada' && prestacion.estados[prestacion.estados.length-1].tipo !== 'auditoria'">{{prestacion.estados[prestacion.estados.length-1].tipo}}</span>
-                                                        <span class="badge badge-danger" *ngIf="prestacion.estados[prestacion.estados.length-1].tipo === 'rechazada'">{{prestacion.estados[prestacion.estados.length-1].tipo}}</span>
+                                                        <span class="badge badge-danger" *ngIf="prestacion.estados[prestacion.estados.length-1].tipo === 'rechazada'">CONTRARREFERIDA</span>
```

Comment:
```suggestion
                                                        <plex-badge type="info" *ngIf="prestacion.estados[prestacion.estados.length-1].tipo === 'rechazada'">CONTRARREFERIDA</plex-badge>
```
Habría que usar `<plex-badge>` en lugar de `<span>` con clases css de bootstrap